lib/validators/calculating_smoking_gun_validator.rb

```diff
@@ -0,0 +1,82 @@
+module Validators
+  class CalculatingSmokingGunValidator < SmokingGunValidator
+    include Validators::Validator
+    def initialize(measures, records, test_id)
+      if measures.length > 1
+        # throw some kind of an error here because we only want to do this for
```

Comment:
I think `add_error` should work in this context, so could we throw an error like "Too many measures provided for a single-measure test" in the initializer? Not sure if we would then want to not validate the first measure, or continue to evaluate it anyways.